Description: Bump version and bug reporting address
Origin: vendor
Forwarded: not-needed
Author: Andrés Roldán <aroldan@debian.org>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2012-05-10

--- valgrind.orig/configure
+++ valgrind/configure
@@ -552,9 +552,9 @@
 # Identity of this package.
 PACKAGE_NAME='Valgrind'
 PACKAGE_TARNAME='valgrind'
-PACKAGE_VERSION='3.7.0'
-PACKAGE_STRING='Valgrind 3.7.0'
-PACKAGE_BUGREPORT='valgrind-users@lists.sourceforge.net'
+PACKAGE_VERSION='3.7.0-Debian'
+PACKAGE_STRING='Valgrind 3.7.0-Debian'
+PACKAGE_BUGREPORT='valgrind-users@lists.sourceforge.net; valgrind@packages.debian.org'
 PACKAGE_URL=''
 
 ac_unique_file="coregrind/m_main.c"
@@ -5279,7 +5279,7 @@
         ARCH_MAX="s390x"
         ;;
 
-     armv7*)
+     arm*)
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: ok (${host_cpu})" >&5
 $as_echo "ok (${host_cpu})" >&6; }
 	ARCH_MAX="arm"
